### PR TITLE
Use the same issuer in token and discovery responses

### PR DESCRIPTION
**Pull request by alecpl**
OIDC clients compare discovered issuer with token's issuer. So, these must be the same.

Mentioned in #23.

### CODE REVIEW

**Comment by alecpl** [comment]
I see that there are no tests for Laravel stuff including DiscoveryController. It means that some tests fail because can't use `url()` without Laravel. I'm not sure what to do with that.

**Comment by alecpl** [comment]
I added forcing of `https://` in Discovery. So, it fixes #23. The issuer is simplified/unified to `'https://' . $_SERVER['HTTP_HOST']` in both places.

File: src/IdTokenResponse.php [hunk 1]
```diff
@@ -55,17 +55,10 @@ protected function getBuilder(
...
-            $issuer = $uri->getScheme() . '://' . $uri->getHost() . ($uri->getPort() ? ':' . $uri->getPort() : '');
-        } else {
-            $issuer = 'https://' . $_SERVER['HTTP_HOST'];
-        }
-
         return $this->config
             ->builder()
             ->permittedFor($accessToken->getClient()->getIdentifier())
-            ->issuedBy($issuer)
+            ->issuedBy('https://' . $_SERVER['HTTP_HOST'])
```
Comment (jeremy379) [inline src/IdTokenResponse.php:61]: I don't think we should force the https here, we could use the default value from laravel and fallback on https only if missing. 
Locally, we may want to work on unsecured connection.

Comment (coffe4u) [reply src/IdTokenResponse.php:61]: To be fair it was `'https://' . $_SERVER['HTTP_HOST']` before https://github.com/jeremy379/laravel-openid-connect/pull/13

File: src/Laravel/DiscoveryController.php [hunk 2]
```diff
@@ -13,8 +14,10 @@ class DiscoveryController
      */
     public function __invoke(Request $request)
     {
+        URL::forceScheme('https'); // for route() calls below
+
         $response = [
-            'issuer' => url('/'),
+            'issuer' => 'https://' . $_SERVER['HTTP_HOST'],
```
Comment (christiaangoossens) [inline src/Laravel/DiscoveryController.php:20]: @jeremy379 This broke my unit test which was calling it without a HTTP_HOST. Maybe a good idea to fallback on both to route('/'), so also for the Id & Access tokens?

Comment (jeremy379) [reply src/Laravel/DiscoveryController.php:20]: Hello
I'm sorry for these issue

Request isn't directly available here, so I choose to provide option on how this should be filled.

Here is a fix MR: https://github.com/jeremy379/laravel-openid-connect/pull/33

**Pull request merged**